test/testutil.py

```diff
@@ -147,3 +148,102 @@ def create_filesystem_customizations(rootfs: str):
     "-v", "/var/lib/containers/storage:/var/lib/containers/storage",
     "--security-opt", "label=type:unconfined_t",
 ]
+
+
+def get_ip_from_default_route():
+    default_route = subprocess.run([
+        "ip",
+        "route",
+        "list",
+        "default"
+    ], check=True, capture_output=True).stdout
+    return default_route.split()[8].decode("utf-8")
+
+
+@dataclasses.dataclass
+class GPGConfig():
+    email: str = "bootc-image-builder@redhat.com"
+    pub_key: str = "/etc/pki/rpm-gpg/RPM-GPG-KEY-booc-image-builder"
```

Comment:
Hm, this will create a key in the real root filesystem, is this intentional? I.e. if I run "sudo pytest" this will modify my real root? Is there a different way maybe?

Reply:
I moved everything to a temporary directory except the lookaside config. We need that in the host system so the sigstore is created by skopeo.